Packs/EDL/Integrations/EDL/README.md

```diff
@@ -1,13 +1,22 @@
 This integration provides External Dynamic List (EDL) as a service for the system indicators (Outbound feed). The feed content can then be sent to 3rd parties for improving their security alignment.
 
+## PAN-OS EDL Management to PAN-OS EDL Service migration steps
+This integration hosts the EDL on the Cortex XSOAR server. Please follow these steps to migrate your EDLs.
+1. Create IOCs in Cortex XSOAR from the EDLs data, which was previously hosted on an external web server. To do so automatically, extract your EDL as a text file from the web server, upload it as a file to the war-room and use the `ExtractIndicatorsFromTextFile` automation. e.g, `!ExtractIndicatorsFromTextFile entryID=<entry_id>` 
+2. Go to the `Indicators` and [filter](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/5-5/cortex-xsoar-admin/manage-indicators/understand-indicators/indicators-page.html#idf15421df-a0e0-41fb-b8d4-07d0f610bbec) to find all of the Indicators you extracted from the text file.
```

Comment:
go to the indicators **page**...
also - how would the user need to filter in order to see all of the relevant indicators?

Reply:
`First Seen` or any other time filter.

Reply:
yeah - i tried this out and saw that there is no indication as to how this indicator was created in the system..
it's actually not that critical, considering it's not the filter the user should be using to maintain the EDL - it's just a way of dumping existing knowledge into XSOAR.

in this case, maybe it's better to rephrase to 'Go to the Indicators page and make sure the indicators were created successfully' - making it a generic saying, without getting into the filtering?